spanner/tests/system/test_system.py

```diff
@@ -728,6 +728,28 @@ def _unit_of_work(transaction, test):
 
         return session, committed
 
+    def test_read_with_multiple_keys_index(self):
+        row_count = 10
+        ddl_create = ['CREATE INDEX name ON contacts(first_name, last_name)']
+        columns = self.COLUMNS[1], self.COLUMNS[2]
+        try:
+            operation = self._db.update_ddl(ddl_create)
```

Comment:
Is there a specific reason not to just add that index to the default DDL in `tests/_fixtures.py`?  That would make this test much simpler / clearer (it would contain only fourstatements and the final assertion).

Reply:
No reason, I just wasn't sure if I should have changed the actual fixture instead of temporary changing it.  
Added it permanently to the fixtures.